internal/string.go

```diff
@@ -14,15 +20,10 @@ func StringsToInterfaces(strings []string) []interface{} {
 }
 
 func Title(name string) string {
-	splits := strings.Split(name, " ")
-	titled := ""
-	for _, v := range splits {
-		if len(v) == 0 {
-			continue
-		}
-		titled += strings.Title(v) + "_"
-	}
-	return strings.TrimRight(titled, "_")
+	name = alphanumeric.ReplaceAllString(name, "")
```

Comment:
this will remove chars that don't match the regex `alphanumeric`, because this function was used for `groups` but not for countries and cities. And there was added a country that contains `'` in its name.

Reply:
yes, there is one country that has `'` in its name and with this change will be removed. 
Would it make sense in command line to have `'`? because it just needs the escape char and might look not very nice, plus doesn't work copy/paste.
I'll try to clarify this

Reply:
Aah, yes. That makes perfect sense now :+1: